.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+
+jobs:
+  build:
+    steps:
+      - checkout
+
+      - run:
+          name: Build Lambda bundle
+          command: npm run build
+
+      - store_artifacts:
+          path: fxa-email-event-proxy.zip
```

Comment:
might be useful to change this name to include the commit or the tag name to avoid misunderstanding